components/neeva/renderer/content_filtering_agent.cc

```diff
@@ -105,6 +113,7 @@ void ContentFilteringAgent::RunScriptsAtDocumentStart(
 
 void ContentFilteringAgent::OnContentFiltered(
     int32_t render_frame_id, mojom::ContentFilterActionPtr action) {
+  // Any actions/communication to mojom ContentFilterService must be ran on the same thread as the agent/service.
```

Comment:
This is a good comment. Please just line wrap to 80 columns. And fix the last part... should say "must be run on the same thread on which `service_` was bound" or something like that.